src/Uno.UI/UI/Xaml/Controls/CalendarDatePicker/CalendarDatePickerFlyout.Android.cs

```diff
@@ -0,0 +1,69 @@
+﻿#if XAMARIN_ANDROID
+using Android.App;
+using Java.Util;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Text;
+using Uno.UI;
+using Uno.UI.Common;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+
+namespace Windows.UI.Xaml.Controls
+{
+    public partial class CalendarDatePickerFlyout : PickerFlyoutBase
+    {
+        private DatePickerDialog _dialog;
+
+        internal protected override void Open()
+        {
+			// Note: Month needs to be -1 since on Android months go from 0-11
+			// http://developer.android.com/reference/android/app/DatePickerDialog.OnDateSetListener.html#onDateSet(android.widget.DatePicker, int, int, int)
+
+			DateTimeOffset tempDate = DateTimeOffset.Now; //  Date.GetValueOrDefault(DateTimeOffset.Now);
+
+            _dialog = new DatePickerDialog(
+                ContextHelper.Current,
+                OnDateSet,
+				tempDate.Year,
+				tempDate.Month - 1,
+				tempDate.Day
+            );
+
+            //Removes title that is unnecessary as it is a duplicate -> http://stackoverflow.com/questions/33486643/remove-title-from-datepickerdialog 
+            _dialog.SetTitle("");
+
+            var minYearCalendar = Calendar.Instance;
+            minYearCalendar.Set(MinDate.Year, MinDate.Month - 1, MinDate.Day, MinDate.Hour, MinDate.Minute, MinDate.Second);
```

Comment:
Won't this set a date that one month before DateTimeOffset.MinValue ?

Reply:
Android uses 0..11 as months.